packages/flutter_tools/templates/ffi_plugin/android.tmpl/build.gradle.tmpl

```diff
@@ -0,0 +1,57 @@
+// We use the Android Gradle Plugin to build the native code with the Android NDK.
```

Comment:
I think if at all possible we should avoid "we" in template comments. This is instantiated into a developer's project, with the expectation that they own, and will edit, this file going forward. The distinction between "we" and "you" in that context is tenuous at best.